content/departments/product-engineering/engineering/cloud/delivery/managed/resume_process.md

```diff
@@ -0,0 +1,114 @@
+# Resumeing a managed instance
+
+This page documents how to resume suspended [managed instances](./index.md). This is useful when Sales team would like to re-engage a customer again where their managed instance was suspended.
+
+Managed instances configuration is tracked in [deploy-sourcegraph-managed].
+
+For basic operations like accessing an instance for these steps, see [managed instances operations](operations.md).
+
+- [General setup](#general-setup)
+- [Sourcegraph resume](#sourcegraph-resume)
+  - [0) Sourcegraph resume setup](#0-sourcegraph-resume-setup)
+  - [1) Prepare terraform module](#1-prepare-terraform-module)
+  - [2) Make database writable](#2-make-database-writable)
+  - [3) Verify instance heath](#3-verify-instance-heath)
+  - [4) Wrapping up](#4-wrapping-up)
+
+## General setup
+
+Managed instances configuration is tracked in [deploy-sourcegraph-managed] - make sure you have the latest revision of this repository checked out. For basic operations like accessing an instance for these steps, see [managed instances operations](operations.md).
+
+First, ensure you have the prerequisites installed and up-to-date:
+
+- [Terraform CLI](https://www.terraform.io/)
+- [Deno CLI](https://deno.land/) (used for scripting upgrades)
+- [Comby CLI](https://comby.dev/) (used for rewriting configuration)
+- [jq](https://stedolan.github.io/jq/)
+
+Many of the following commands in this guide, as well as the commands [operations](./operations.md), use environment variables. Export the appropriate values for the upgrade so you don't lose track:
+
+```sh
+# name of customer deployment (should match folder)
+export CUSTOMER=<customer_or_instance_name>
+
+# the previous actvie deployment, either `red` or `black`
+# this can be determined by obtaining the last snapshot name by running `gcloud compute snapshots list`
+# `default-red-data-disk-snapshot--upgrade-from-<>` -> `red`
+# `default-black-data-disk-snapshot--upgrade-from-<>` -> `black`
+export CURRENT_DEPLOYMENT="[red|black]"
```

Comment:
In the operations guide we embed the gcloud command into a copy-pastable form, like this:
```
export OLD_DEPLOYMENT=$(gcloud compute instances list --project=sourcegraph-managed-${CUSTOMER} | grep -v "executors" | awk 'NR>1 { if ($1 ~ "-red-") print "red"; else print "black"; }')
```

Is there a way we can do the same here? This is a rare situation so not a huge deal if we've introduced a manual step, just curious if we can make it better.